shader/fine.wgsl

```diff
@@ -239,7 +239,7 @@ fn main(
                 let rad = read_rad_grad(cmd_ix);
                 for (var i = 0u; i < PIXELS_PER_THREAD; i += 1u) {
                     let my_xy = vec2(xy.x + f32(i), xy.y);
-                    // TODO: can hoist y, but for now stick to piet-gpu
+                    // TODO: can hoist y, but for now stick to vello
```

Comment:
This is also misleading, as I was referring to the GLSL version. Probably clearest to change it to "the GLSL version" to avoid confusion.

Reply:
Roger that